java/src/shared/model/game/TurnTracker.java

```diff
@@ -82,6 +82,9 @@ public int nextTurn() throws Exception {
         } else {
             currentTurn++;
             currentTurn %= NUM_PLAYERS;
+            if(phase == Phase.PLAYING){
+                phase = Phase.ROLLING;
+            }
```

Comment:
He was with me when I did it
On Mar 21, 2016 9:36 PM, "Derek Argueta" <notifications@github.com> wrote:

> In java/src/shared/model/game/TurnTracker.java
> <https://github.com/derekargueta/Settlers-of-Catan/pull/138#discussion_r56933428>
> :
>
> > @@ -82,6 +82,9 @@ public int nextTurn() throws Exception {
> >          } else {
> >              currentTurn++;
> >              currentTurn %= NUM_PLAYERS;
> > +            if(phase == Phase.PLAYING){
> > +                phase = Phase.ROLLING;
> > +            }
>
> @yeyyosing <https://github.com/yeyyosing> @dannyharding10
> <https://github.com/dannyharding10> please verify that this is desired
> behavior.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/derekargueta/Settlers-of-Catan/pull/138/files/df673b6ee34e3d75d9101151f57bd3ddb2884e5c#r56933428>
>